tempesta_fw/http_msg.c

```diff
@@ -321,6 +321,14 @@ tfw_http_msg_hdr_close(TfwHttpMsg *hm, int id)
 done:
 	*h = hm->parser.hdr;
 
+	/*
+	 * We must distinguish headers opened after the body from generic
+	 * headers, so mark them by setting @eolen value to 0. It helps us to
+	 * prevent saving them to the cache (FIXME: #491).
+	 */
+	if (hm->body.flags & TFW_STR_COMPLETE)
+		tfw_str_set_eolen(h, 0);
```

Comment:
Standard doesn't say anything about treating trailer headers in different manner than usual header, so why should we do this?